### PR TITLE
Updated totalRow within ResultSet

### PR DESCRIPTION
**Pull request by nuwe1**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**

This PR updates the types in @cubejs-client/core by  addressing the issue https://github.com/cube-js/cube.js/issues/589

### CODE REVIEW

**Comment by nuwe1** [comment]
@rpaik  Could you assign this to me? Also, I am not quite sure about how to find `totalRow` Do you have any thoughts or ideas.

**Comment by rpaik** [comment]
Thanks @nuwe1 😄 Is there an issue associated with this?

**Comment by nuwe1** [comment]
> Thanks @nuwe1 😄 Is there an issue associated with this?

Yes @rpaik  I just update the description to reflect that but you can quickly access it right here https://github.com/cube-js/cube.js/issues/589

**Comment by rpaik** [comment]
@hassankhan can you take a quick look at this?

**Review by hassankhan — APPROVED** [review]
Looks good to me but I think we need @vasilev-alex's review too 👍

**Comment by vasilev-alex** [comment]
@nuwe1 thanks for the PR! Could you also update the types in the `packages/cubejs-client-core/index.d.ts` file?

**Review by vasilev-alex — CHANGES_REQUESTED** [review]
@nuwe1 thanks for the PR! Could you also update the types in the `packages/cubejs-client-core/index.d.ts` file?

**Comment by nuwe1** [comment]
@vasilev-alex   I have updated the types in the packages/cubejs-client-core/index.d.ts, Could you please give it another look?

File: packages/cubejs-client-core/index.d.ts [hunk 1]
```diff
@@ -560,6 +560,7 @@ declare module '@cubejs-client/core' {
      * @returns An array of columns
      */
     tableColumns(pivotConfig?: PivotConfig): TableColumn[];
+    totalRow(pivotConfig?: PivotConfig):{ [key: string]: number };
```
Comment (vasilev-alex) [inline packages/cubejs-client-core/index.d.ts:563]: As `totalRow` returns the first item from the `chartPivot` array I believe the return type should be `ChartPivotRow`

Comment (nuwe1) [reply packages/cubejs-client-core/index.d.ts]: I got it 😄.  Let me update that right away

Comment (nuwe1) [reply packages/cubejs-client-core/index.d.ts]: Updated !!! Could you please take another look?

**Comment by rpaik** [comment]
Congratulations @nuwe1 on having your first PR merged in the Cube.js community 🎉 🎊 🎆

**Pull request merged**